bundles/net.tourbook.export/src/net/tourbook/export/fit/FitExporter.java

```diff
@@ -237,8 +242,12 @@ private List<EventMesg> createEventMessages(final DateTime startTime,
             final EventMesg eventMesgStop = new EventMesg();
             final Date pausedTime_Start_Date = Date.from(Instant.ofEpochMilli(pausedTime_Start[index]));
             eventMesgStop.setTimestamp(new DateTime(pausedTime_Start_Date));
-            // By default: eventData == 1: auto-stop
-            eventMesgStop.setData(1L);
+            /**
+             * eventData == 0: user stop<br>
+             * eventData == 1: auto-stop
+             */
```

Comment:
You copied my comment 😄 

![image](https://github.com/mytourbook/mytourbook/assets/1283445/c5701273-0a1e-42e3-86c4-c288f098db59)

Reply:
Yes, it was well written so I didn't want to "reinvent the wheel" 😄